data/mods/Craft_Gunpowder/cgp_ranged.json

```diff
@@ -24,7 +24,7 @@
     "clip_size": 1,
     "reload": 250,
     "loudness" : 150,
-    "valid_mod_locations": [[ "muzzle", 1 ],[ "stock", 1 ]]
+    "valid_mod_locations": [ [ "accessories", 2 ], [ "muzzle", 1 ], [ "stock", 1 ] ]
   },
   {
     "id": "rifle_38",
```

Comment:
Again you should consider `copy-from` (can be a separate PR). The mod becomes more robust the lest duplicate/redundant entries it contains